frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -124,79 +177,37 @@ function FileManagerCollection:onMenuHold(item)
     return true
 end
 
-function FileManagerCollection:MenuSetRotationModeHandler(rotation)
-    if rotation ~= nil and rotation ~= Screen:getRotationMode() then
-        UIManager:close(self._manager.coll_menu)
-        if self._manager.ui.view and self._manager.ui.view.onSetRotationMode then
-            self._manager.ui.view:onSetRotationMode(rotation)
-        elseif self._manager.ui.onSetRotationMode then
-            self._manager.ui:onSetRotationMode(rotation)
-        else
-            Screen:setRotationMode(rotation)
-        end
-        self._manager:onShowColl()
-    end
-    return true
-end
-
-function FileManagerCollection:onShowColl(collection_name)
-    collection_name = collection_name or ReadCollection.default_collection_name
-    self.coll_menu = Menu:new{
-        ui = self.ui,
-        covers_fullscreen = true, -- hint for UIManager:_repaint()
-        is_borderless = true,
-        is_popout = false,
-        -- item and book cover thumbnail dimensions in Mosaic and Detailed list display modes
-        -- must be equal in File manager, History and Collection windows to avoid image scaling
-        title_bar_fm_style = true,
-        title_bar_left_icon = "appbar.menu",
-        onLeftButtonTap = function() self:showCollDialog() end,
-        onMenuChoice = self.onMenuChoice,
-        onMenuHold = self.onMenuHold,
-        onSetRotationMode = self.MenuSetRotationModeHandler,
-        _manager = self,
-        collection_name = collection_name,
-    }
-    self.coll_menu.close_callback = function()
-        if self.files_updated then
-            if self.ui.file_chooser then
-                self.ui.file_chooser:refreshPath()
-            end
-            self.files_updated = nil
-        end
-        UIManager:close(self.coll_menu)
-        self.coll_menu = nil
-    end
-    self:updateItemTable()
-    UIManager:show(self.coll_menu)
-    return true
-end
-
 function FileManagerCollection:showCollDialog()
     local coll_dialog
     local buttons = {
         {{
-            text = _("Sort favorites"),
+            text = _("Collections"),
+            callback = function()
+                UIManager:close(coll_dialog)
+                self.coll_menu.close_callback()
+                self:onShowCollList()
+            end,
+        }},
+        {}, -- separator
+        {{
+            text = _("Sort collection"),
```

Comment:
I'd assume it's referring to sorting[1] books within a collection, not to sorting the collections themselves?

[1] I was expecting something else when I saw the word sort. ;-) I suspect that if we do change sort to arrange over in #11678 there probably aren't any occurrences of *sort* where the same reasoning doesn't apply. I.e., that arranging doesn't imply something more defined like sorting by alphabetical order.

Reply:
the word "sort" has an undertone of 'prescribed sequence', "arrange" is more liberal, on a whim sort of way.

I choose 'arrange' because the user chooses whatever he/she wants and in what order. So if its referring to 'sorting books within a collection", 'Arrange' would work better.